Makefile

```diff
@@ -15,205 +15,182 @@
-#add dependency on custom makefile filename
-dep/%.d: %.c Makefile
-
-listvariants:
-	@echo VARIANTS COIN cardano_ada
-
-# part of CI
-analyze: clean
-	scan-build --use-cc=clang -analyze-headers -enable-checker security -enable-checker unix -enable-checker valist -o scan-build --status-bugs make default
-
-##############
-#   Load     #
-##############
+########################################
+#     Application custom permissions   #
+########################################
+# See SDK `include/appflags.h` for the purpose of each permission
+#HAVE_APPLICATION_FLAG_DERIVE_MASTER = 1
+#HAVE_APPLICATION_FLAG_GLOBAL_PIN = 1
+HAVE_APPLICATION_FLAG_BOLOS_SETTINGS = 1
+#HAVE_APPLICATION_FLAG_LIBRARY = 1
+
+########################################
+# Application communication interfaces #
+########################################
+ENABLE_BLUETOOTH = 1
+#ENABLE_NFC = 1
+
+########################################
+#         NBGL custom features         #
+########################################
+ENABLE_NBGL_QRCODE = 1
+#ENABLE_NBGL_KEYBOARD = 1
+#ENABLE_NBGL_KEYPAD = 1
+
+########################################
+#          Features disablers          #
+########################################
+# These advanced settings allow to disable some feature that are by
+# default enabled in the SDK `Makefile.standard_app`.
+#DISABLE_STANDARD_APP_FILES = 1
+#DISABLE_DEFAULT_IO_SEPROXY_BUFFER_SIZE = 1 # To allow custom size declaration
+#DISABLE_STANDARD_APP_DEFINES = 1 # Will set all the following disablers
+#DISABLE_STANDARD_SNPRINTF = 1
+#DISABLE_STANDARD_USB = 1
+#DISABLE_STANDARD_WEBUSB = 1
+#DISABLE_STANDARD_BAGL_UX_FLOW = 1
+
+########################################
+#       Additional configuration       #
+########################################
+
+# USB U2F
+DEFINES += HAVE_U2F HAVE_IO_U2F U2F_PROXY_MAGIC=\"ADA\"
+SDK_SOURCE_PATH  += lib_u2f
+
+DEFINES += HAVE_UX_LEGACY
+
+# Protect against stack overflows
+DEFINES += HAVE_BOLOS_APP_STACK_CANARY
 
-NANOS_ID = 1
+# mnemonic and PIN for testing on a physical device
 WORDS = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about"
 PIN = 5555
 
-APP_LOAD_PARAMS =--appFlags 0x200 --curve ed25519 --path "44'/1815'" --path "1852'/1815'" --path "1853'/1815'" --path "1854'/1815'" --path "1855'/1815'" --path "1694'/1815'"
-APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
+# based in part on https://interrupt.memfault.com/blog/best-and-worst-gcc-clang-compiler-flags
+WERROR   := -Werror=return-type -Werror=parentheses -Werror=format-security
+CFLAGS   += -Wuninitialized -fno-common $(WERROR)
 
-load:
-	python -m ledgerblue.loadApp $(APP_LOAD_PARAMS)
+########################################
+#          Additional targets          #
+########################################
 
-delete:
-	python -m ledgerblue.deleteApp $(COMMON_DELETE_PARAMS)
+# code style
+format:
+	astyle --options=.astylerc "src/*.h" "src/*.c" --exclude=src/glyphs.h --exclude=src/glyphs.c --ignore-exclude-errors
```

Comment:
glyphs.c and glyphs.h are not generated in src anymore.
You could make a CI check of the coding style to make sure code stay formatted.